conseil-api/src/test/scala/tech/cryptonomic/conseil/api/routes/platform/data/ethereum/EthereumDataOperationsTest.scala

```diff
@@ -133,6 +132,19 @@ class EthereumDataOperationsTest
         }
       }
 
+      "return proper token balances, while fetching all token balances" in {
```

Comment:
I would encourage you to add similar tests to your temporal table like I did for `accounts_history`: https://github.com/Cryptonomic/Conseil/blob/103c5f7b5db2b1d28ab0c4050f306fdf52b20585/conseil-api/src/test/scala/tech/cryptonomic/conseil/api/routes/platform/data/tezos/TezosDataOperationsTest.scala#L2312

Reply:
great, thanks!